### PR TITLE
Update documentations for Kubeflow Pipelines marketplace

### PR DESCRIPTION
**Pull request by IronPan**
This change update the documentations that are visible to the end user when they visiting marketplace page.
/assign @sarahmaddox

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2125)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Thanks @IronPan! I don't have bandwidth to review this PR at the moment. I'll email you with some suggestions about reviewers.

File: manifests/gcp_marketplace/schema.yaml [hunk 1]
```diff
@@ -101,33 +101,59 @@ properties:
     x-google-marketplace:
       type: NAMESPACE
   serviceAccountCredential:
-    title: Service Account credentials used to call other GCP services.
+    title: |-
+      Service Account credentials used to call other GCP services.
```
Comment (rmgogogo) [inline manifests/gcp_marketplace/schema.yaml]: wonding whether we call it "GCP Service Account credentials used ...". 
As talked last time, we have two types of service account, one is K8s and other is GCP.

File: manifests/gcp_marketplace/schema.yaml [hunk 2]
```diff
@@ -101,33 +101,59 @@ properties:
...
     description: |-
-      To be able to call other GCP services, we need to be
-      authenticated. This field is used to store the content of the service account
+      To be able to call other GCP services from the pipeline, we need to be authenticated.
+      This field is used to store the content of the service account
       JSON file. It can be encoded using base64 instead of messing with JSON format.
+      To get the encoded credential, run
+      $ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64
```
Comment (rmgogogo) [inline manifests/gcp_marketplace/schema.yaml]: not sure whether the special characters here '$', '&' will be escaped by MKP UI, we may need to double check it manually after another staging in MKP

Comment (IronPan) [reply manifests/gcp_marketplace/schema.yaml]: i verified it and should be ok.

File: manifests/gcp_marketplace/schema.yaml (hunk 1 above)
Comment (IronPan) [reply manifests/gcp_marketplace/schema.yaml]: good point. updated.

**Pull request merged**